### PR TITLE
Use more constants, isolate SV6 limits further

### PR DESCRIPTION
**Pull request by Gymnasiast**
This further decouples SV6 limits from OpenRCT2 limits.

### CODE REVIEW

File: src/openrct2/ride/ride.c [hunk 1]
```diff
@@ -2032,7 +2032,7 @@ static void ride_update(sint32 rideIndex)
         ride->num_customers_timeout = 0;
 
         // Shift number of customers history, start of the array is the most recent one
-        memmove(ride->num_customers + 1, ride->num_customers, 9 * sizeof(*(ride->num_customers)));
+        memmove(ride->num_customers + 1, ride->num_customers, sizeof(ride->num_customers) - 1);
```
Comment (IntelOrca) [inline src/openrct2/ride/ride.c]: This isn't right
`sizeof(ride->num_customers) = 10 * 2 = 20`
`sizeof(ride->num_customers) - 1 = 19`
You only want to move `9 * sizeof(ride->num_customers)`

Comment (IntelOrca) [inline src/openrct2/ride/ride.c]: I would personally use a for loop as its clearer for the developer. The compiler can turn it into `memmove`.

Comment (Gymnasiast) [inline src/openrct2/ride/ride.c]: Like this:
```
for (sint32 i = NUM_CUSTOMERS_SIZE; i > 0; i--)
{
    ride->num_customers[i] = ride->num_customers[i - 1];
}
```
?

Comment (IntelOrca) [inline src/openrct2/ride/ride.c]:
```
for (sint32 i = NUM_CUSTOMERS_SIZE - 1; i > 0; i--)
{
    ride->num_customers[i] = ride->num_customers[i - 1];
}
```

**Pull request merged**